commit 04ec365e3fdf136ba5f9053b02fb6c3368a22e83
Author: Ujwal Kundur <ujwal.kundur@gmail.com>
Date:   Sat Feb 15 16:21:07 2025 +0530

    Documentation: fix doc link to fault-injection.rst
    
    Fix incorrect reference to fault-injection docs
    
    Link: https://lkml.kernel.org/r/20250215105106.734-1-ujwal.kundur@gmail.com
    Signed-off-by: Ujwal Kundur <ujwal.kundur@gmail.com>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index 1af972a92d06..35796c290ca3 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -2103,7 +2103,7 @@ config FAIL_SKB_REALLOC
 	  reallocated, catching possible invalid pointers to the skb.
 
 	  For more information, check
-	  Documentation/dev-tools/fault-injection/fault-injection.rst
+	  Documentation/fault-injection/fault-injection.rst
 
 config FAULT_INJECTION_CONFIGFS
 	bool "Configfs interface for fault-injection capabilities"